gladier_tools/posix/tar.py

```diff
@@ -0,0 +1,47 @@
+from gladier import GladierBaseTool
+
+
+def tar(data):
+    import os
+    import tarfile
+    tar_input = data['tar_input']
```

Comment:
maybe we need to check if it is a single file or folder or file list?
if file list it will need to be expanded before passing to the cmd